tests/queries/test_db_returning.py

```diff
@@ -25,8 +25,7 @@ def test_insert_returning_non_integer(self):
         self.assertTrue(obj.created)
         self.assertIsInstance(obj.created, datetime.datetime)
 
-    @skipUnlessDBFeature('can_return_multiple_columns_from_insert')
-    def test_insert_returning_multiple(self):
```

Comment:
Please revert this change. We can get an unsupported combination of DB features in the future, so it's good to keep it.